scripts/fixup_headers.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python3
+
+# This script checks and can optionally update FENIX source files.
+# You should always run this script without the "-u" option
+# first to make sure there is a clean dry run of the files that should
+# be updated
+# This is based on a script of the same name in the MOOSE Framework:
+# https://github.com/idaholab/moose/blob/master/framework/scripts/fixup_headers.py
+
+import os, string, re, shutil
+from optparse import OptionParser
+
+global_dir_ignores = ['contrib', '.svn', '.git', 'crane', 'moose', 'squirrel']
+global_file_ignores = ['moosedocs.py']
+
+unified_header = """\
+//* This file is part of FENIX: Fusion ENergy Integrated multiphys-X,
+//* A multiphysics application for modeling plasma facing components
+//* https://github.com/idaholab/fenix
+//*
+//* FENIX is powered by the MOOSE Framework
+//* https://www.mooseframework.inl.gov
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+//*
+//* Copyright 2024, Battelle Energy Alliance, LLC
+//* ALL RIGHTS RESERVED
+//*"""
+
+python_header = """\
+//* This file is part of FENIX: Fusion ENergy Integrated multiphys-X,
+//* A multiphysics application for modeling plasma facing components
+//* https://github.com/idaholab/fenix
+//*
+//* FENIX is powered by the MOOSE Framework
+//* https://www.mooseframework.inl.gov
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+//*
+//* Copyright 2024, Battelle Energy Alliance, LLC
+//* ALL RIGHTS RESERVED
+//*"""
```

Comment:
Python uses a different comment symbol.

```suggestion
#* This file is part of FENIX: Fusion ENergy Integrated multiphys-X,
#* A multiphysics application for modeling plasma facing components
#* https://github.com/idaholab/fenix
#*
#* FENIX is powered by the MOOSE Framework
#* https://www.mooseframework.inl.gov
#*
#* Licensed under LGPL 2.1, please see LICENSE for details
#* https://www.gnu.org/licenses/lgpl-2.1.html
#*
#* Copyright 2024, Battelle Energy Alliance, LLC
#* ALL RIGHTS RESERVED
#*"""
```